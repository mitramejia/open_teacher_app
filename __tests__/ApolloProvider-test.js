import React from 'react';
import renderer from 'react-test-renderer';
import { MockComponent } from 'react-native/jest/mockComponent';
import { ApolloProvider } from 'react-apollo';
import { TestUtils } from 'react-dom/test-utils';

it('renders the Apollo Provider component', () => {
  const client = {};
  const mockComponent = React.createElement('Text');
  const tree = renderer
    .create(<ApolloProvider client={client} children={mockComponent} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
