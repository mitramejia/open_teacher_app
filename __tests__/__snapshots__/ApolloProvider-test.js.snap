// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Apollo Provider component 1`] = `<Text />`;
